### DISCUSSION TITLE
Mesh Size Problem

### DISCUSSION THREAD

**Discussion by Ali141272** [neperfepx/neper#459] [Q&A] [answered]
Hello Respected Sir,
Sir, I have built the model in Neper but I am facing one issue, which is related to mesh size.  I need your suggestion, on how to fix this problem.
I highlighted the problem in the model, please find the enclosed attached file of the model.
I hope for your kind consideration and response.
Yours sincerely 
Ali
![Ali model](https://user-images.githubusercontent.com/102291665/166152138-797a7875-5a7d-4e7b-a170-a97a87f42b34.JPG)

**Comment by mkasemer** [comment, marked as answer]
@Ali141272 - in the absence of any minimal working example, which is necessary to properly diagnose an issue, I can only assume that you have not turned on regularization during tessellation. Please consult the documentation for further guidance, but I suggest creating the tessellation with tessellation regularization enabled (`-reg 1`).

**Comment by Ali141272** [comment]
Hello, sir Thank you so much for your time and response. Sir are you talking about the below-mentioned command
Generate a Voronoi tessellation containing 100 cells and apply regularization:

$ neper -T -n 100 -reg 1

Sir this is my commands
$ neper -T -n 40
$ neper -T -n 40 -id 2
$ neper -T -n 40 -domain "square(0.025,0.05)" -dim 2
$ neper -M n40-id1.tess
$ neper -V n40-id1.tess -print a
$ neper -V n40-id1.msh -print b

I hope for your kind response.
Yours sincerely, 
Ali

**Reply by mkasemer** [reply]
@Ali141272 - I do not understand why you have three separate `-T` commands, but it is evident that none of them utilize the regularization option, as you should in your top command (`-reg 1`). Try applying this to your tessellation generation commands.

**Comment by Ali141272** [comment]
Hello, sir thank you so much for your time and response. Sir are you talking about the below-mentioned command   
   -    
Generate a Voronoi tessellation containing 100 cells and apply regularization:
   $ neper -T -n 100 -reg 1
Sir this is my coding

$ neper -T -n 40
$ neper -T -n 40 -id 2
$ neper -T -n 40 -domain"square(0.025,0.05)" -dim 2

$ neper -M n40-id1.tess

$ neper -V n40-id1.tess -print a


$ neper -V n40-id1.msh -print b


I hope for your kind response
Sincerely 
Ali


    On Sunday, May 1, 2022, 11:36:07 PM GMT+8, Matthew Kasemer ***@***.***> wrote:  
 
 


@Ali141272 - in the absence of any minimal working example, which is necessary to properly diagnose an issue, I can only assume that you have not turned on regularization during tessellation. Please consult the documentation for further guidance, but I suggest creating the tessellation with tessellation regularization enabled (-reg 1).
